Projects/GLU/GLU01/GLU01-AWS100.md

```diff
@@ -0,0 +1,38 @@
+# GLU02-AWS100 - Introduction to SNS (Simple Notification Service)
+
+## Cloud Service Provider
+- Amazon Web Services
+
+## Difficulty
+- Level 100 (Introductory)
+
+## Project's Author(s)
+
+- [Edward Allen Mercado](https://twitter.com/edwardmercado_)
+
+## Objectives
+
+### You need to complete the following:
+
+- Create a SNS Topic
+- Subscribe to a SNS Topic
+- Receive email notification from the SNS Topic created
+
+### You need to answer the following:
+
+- What is SNS?
+- How Pub/Sub notifications work?
+- What are the different endpoints that SNS can send notification?
+
+## References
+
+- [AWS SNS Official Documentation](https://docs.aws.amazon.com/sns/latest/dg/welcome.html)
```

Comment:
**Add**
- [Setting Up Amazon SNS Notifications](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/US_SetupSNS.html)
- [Amazon SNS (from AWS) - The Ultimate Guide](https://www.serverless.com/amazon-sns)
- [What is Amazon SES and SNS in AWS?](https://intellipaat.com/blog/what-is-amazon-ses-sns-in-aws/)

Reply:
Thank you! added them